File: server/src/main/java/com/example/schoolsite/services/SheduleService.java
package com.example.schoolsite.services;

import com.example.schoolsite.dto.SheduleDTO;
import com.example.schoolsite.exception.ResourceNotFoundException;

import java.util.List;

public interface SheduleService {

    List<SheduleDTO> getScheduleByIdAndDate(Long userId, String date)
            throws ResourceNotFoundException;

    List<SheduleDTO> getScheduleDTOByIdAndDate(Long userId, String date);
}

File: server/src/main/java/com/example/schoolsite/services/SheduleService.kt
package com.example.schoolsite.services

import com.example.schoolsite.dto.SheduleDTO

interface SheduleService {
    @Throws(ResourceNotFoundException::class)
    fun getScheduleByIdAndDate(userId: Long?, date: String?): List<SheduleDTO?>?
    fun getScheduleDTOByIdAndDate(userId: Long?, date: String?): List<SheduleDTO?>?
}